### DISCUSSION TITLE
Error "To Early at /api/plugins/sso/acs/" with SALM login

### DISCUSSION THREAD

**Discussion by chrisho95** [netbox-community/netbox#16358] [Help Wanted!] [answered]
Hello,

when I try to connect via SSO (saml) to Netbox (version 4.0.3), I get the following error.

I installed the following plugins:

    django3-auth-saml2 (last version)
    netbox-plugin-auth-saml2 (version 3.0)

![image](https://github.com/netbox-community/netbox/assets/82440344/5166add8-4759-48fd-bcee-f31189e20149)

I can connect directly without going through SSO.

Can anyone help me ?

Br,

Chris.

**Comment by candlerb** [comment, marked as answer]
At a guess, I'd say you need to check whether your system clock is synchronized accurately.

**Reply by chrisho95** [reply]
Hi,

Indeed, the time of the VM that hosts Keycloak is ahead of the VM that hosts my Netbox instance. Thanks for your help.

I'll look into this date issue.

Br,

Chris.
